src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {
+    private String color;
+    private String name;

```

Comment:
You actually don't need name here and in every type of figure also

Reply:
I need variable 'name' for calling it in Figure constructor - for creating method 'getDefoultFigure' - where is parameter 'name'